mxnet-bot/EmailBot/EmailBot.py

```diff
@@ -0,0 +1,382 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import print_function
+from collections import defaultdict
+from botocore.exceptions import ClientError
+from botocore.vendored import requests
+from email.mime.multipart import MIMEMultipart
+from email.mime.image import MIMEImage
+from email.mime.text import MIMEText
+import datetime
+import operator
+import boto3
+import datetime
+import logging
+import os
+import re
+
+logging.basicConfig(level=logging.INFO)
+
+
+class EmailBot:
+
+    def __init__(self, img_file="/tmp/img_file.png", sla=5,
+                 github_user = os.environ.get("github_user"),
+                 github_oauth_token = os.environ.get("github_oauth_token"),
+                 repo = os.environ.get("repo"),
+                 sender = os.environ.get("sender"),
+                 recipients = os.environ.get("recipients"),
+                 aws_region = os.environ.get('aws_region'),
```

Comment:
is AWS_DEFAULT_REGION not enough? I usually never specify the region for boto3. Just curious.

Reply:
Well, it doesn't hurt if we have this option. sometimes I use services in us-east-1 and sometimes us-west-2

Reply:
no problem is just that setting the env variable AWS_DEFAULT_REGION is usually enough for this.